src/main/java/org/jenkinsci/plugins/gitclient/RemoteGitImpl.java

```diff
@@ -56,7 +56,7 @@
         return (hudson.plugins.git.IGitAPI)proxy;
     }
 
-    private Object readResolve() {
+    protected Object readResolve() {
```

Comment:
this is package private so i doubt it's inherited by anything, but the change shouldn't hurt anyone